src/view/RoomsView.java

```diff
@@ -34,14 +35,16 @@ public class RoomsView implements PropertyChangeListener {
   private JLabel roomNameLabel2;
   private JTextField emailTextField;
   private JButton searchButton;
+  private JButton settingsButton;
```

Comment:
Call this `roomsSettingsButton` since we already have a settings page for the user

Reply:
Even though it isn't in this class, I think it would be clearer